app/(public)/components/paragraph/sul-contact-card/manual-fields-card.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+
+import Link from "next/link";
+import Image from "next/image";
+import {ClockIcon, EnvelopeIcon, MapPinIcon, PhoneIcon } from "@heroicons/react/24/outline";
+import Conditional from "@/components/utils/conditional";
+import {useResizeDetector} from "react-resize-detector";
+import { ContactCardParagraph } from "@/lib/drupal/drupal";
+
+const ManualFieldsCard = ({paragraph}: { paragraph: ContactCardParagraph }) => {
+  const {width, ref} = useResizeDetector();
+
+  const contactImageUrl = paragraph.sul_contact__image?.field_media_image?.image_style_uri?.breakpoint_md_2x
+
+  let contactImage
+
+  if (contactImageUrl) {
+    contactImage = <Image
+      className="su-object-cover su-object-center su-static"
+      src={contactImageUrl}
+      alt={paragraph.sul_contact__image?.field_media_image?.resourceIdObjMeta?.alt ?? ''}
+      fill={true}
+    />
+  }
+
+  return (
+    <>
+      <div ref={ref} className={"su-flex su-w-full su-basefont-23 su-leading-display su-shadow-md su-border-0 su-rounded " + ((width && width < 768) && " su-flex-col")}>
+        <div className={"su-overflow-hidden su-aspect-[16/9] su-relative " + ((width && width > 767) && " su-w-1/2")}>
+          {contactImage}
+        </div>
+        
+        <div className={"card-body su-items-start su-rs-px-2 su-rs-py-4 su-bg-black-true " + ((width && width > 767) && " su-w-1/2")}>
+          <div className="su-leading-display su-text-18 su-pt-0 su-font-normal ">
+            <Conditional showWhen={paragraph.sul_contact__title}>
+              <h2 className="su-type-3 su-rs-mb-1 su-text-white">{paragraph.sul_contact__title}</h2>
+            </Conditional>
```

Comment:
Can you put a empty line above and below each `<Conditional>` component. It will make it more readable.